test/rebuild.ts

```diff
@@ -77,6 +77,10 @@ describe('rebuilder', () => {
         await expectNativeModuleToBeRebuilt(testModulePath, 'farmhash');
       });
 
+      it('should have rebuilt top level prod dependencies that are using prebuild + napi', async () => {
+        await expectNativeModuleToBeRebuilt(testModulePath, 'node-hid');
+      });
+
```

Comment:
I took a quick look, `farmhash` uses N-API so that case is already covered. I think what I'm more interested in, in terms of integration tests, is when farmhash is "rebuilt", it doesn't hit the `node-gyp` codepath.

Reply:
Right, got it!